newIDE/app/src/ResourcesList/FileToCloudProjectResourceUploader.js

```diff
@@ -203,7 +246,12 @@ export const FileToCloudProjectResourceUploader = ({
               onChange={event => {
                 const files = [];
                 for (let i = 0; i < event.currentTarget.files.length; i++) {
-                  files.push(event.currentTarget.files[i]);
+                  if (
+                    !shouldValidateFilePostPicking ||
+                    validateFilePostPicking(event.currentTarget.files[i])
+                  ) {
+                    files.push(event.currentTarget.files[i]);
+                  }
```

Comment:
Do users know why some files are excluded?

Reply:
You mean "Will they know why a file has been excluded from their selection if the extension does not match?" In that case, no. I'll see if I can make it clear.

Reply:
I added a message to inform the user: 
<img width="453" alt="image" src="https://github.com/4ian/GDevelop/assets/32449369/9705a449-b4e4-46d5-a30d-5239b8994fe9">